src/server.js

```diff
@@ -469,6 +479,7 @@ function getClientConfiguration() {
 	config.version = pkg.version;
 	config.gitCommit = Helper.getGitCommit();
 	config.themes = themes.getAll();
+	config.stylesheets = packages.getStylesheets();
```

Comment:
This is not needed.

Reply:
Oh yeah, added this in when testing how the new UI worked. Forgot to remove it.